samples/outlook/85-tokens-and-service-calls/basic-rest-cors.yaml

```diff
@@ -1,7 +1,7 @@
 order: 5
 id: outlook-tokens-and-service-calls-basic-rest-cors
 name: Get a message using REST
-description: 'Gets a message using REST without any back-end code.'
```

Comment:
why does this keep on changing :-)  are we using different versions of Yarn or something? I installed latest very recently...

Reply:
not sure. I'm using 1.12.3

Reply:
1.19.1 here